docs/content/dagster-plus/managing-deployments/deployment-settings-reference.mdx

```diff
@@ -228,25 +228,3 @@ non_isolated_runs:
     </ul>
   </ReferenceTableItem>
 </ReferenceTable>
-
-### Auto-materialize
```

Comment:
@OwenKephart I removed this because I assumed it would be axed alongside other AMP stuff. Can you confirm?

Reply:
correct!